ocf_datapipes/training/common.py

```diff
@@ -1265,12 +1265,23 @@ def create_t0_and_loc_datapipes(
     return location_pipe, t0_datapipe
 
 
-def potentially_coarsen(xr_data: xr.Dataset):
-    """Coarsen the data, if it is separated by 0.05 degrees each"""
+def potentially_coarsen(xr_data: xr.Dataset, coarsen_to_deg: float = 0.1):
+    """
+    Coarsen the data, if it is separated by 0.05 degrees each
```

Comment:
I guess this description of the function mentioning 0.05 is no longer strictly true since it now depends on the initial degree separation and the `coarsen_to_deg` parameter

Reply:
thanks, ill update this